setup.py

```diff
@@ -1,11 +1,25 @@
-from distutils.core import setup
+from distutils.command.build import build
+from setuptools import setup
+from setuptools.command.install import install as _install
 
+import unittest
```

Comment:
Please move this to the start of the file. Import order is:

* Complete module imports
* submodule imports